Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -453,9 +453,11 @@ - (void)selectWholeMonth:(BPKSimpleDate *)month {
 - (BOOL)isWholeMonthButtonEnabledForMonth:(BPKSimpleDate *)month {
```

Comment:
Could you please add a test for this logic? 
Not 100% sure if there was a test before, but it feels like we have hit some issues with 'wholeMonth' before.